street_network/src/render.rs

```diff
@@ -65,54 +65,77 @@ impl StreetNetwork {
         Ok(output)
     }
 
-    /// Generates a more detailed GeoJSON rendering of roads and intersections.
-    pub fn to_detailed_geojson(&self, timer: &mut Timer) -> Result<String> {
+    /// Generates a polygon per lane, with a property indicating type.
+    pub fn to_lane_polygons_geojson(&self, timer: &mut Timer) -> Result<String> {
         // TODO InitialMap is going away very soon, but we still need it
         let initial_map = crate::initial::InitialMap::new(self, timer);
```

Comment:
Oh and this starts to get expensive, since all of the layer functions here call it. I need to finish #2 finally.